insights/parsers/tests/test_parsers_module.py

```diff
@@ -1,8 +1,8 @@
 import pytest
+
 from collections import OrderedDict
-from insights.parsers import split_kv_pairs, unsplit_lines, parse_fixed_table
-from insights.parsers import calc_offset, optlist_to_dict, keyword_search
-from insights.parsers import parse_delimited_table, ParseException, SkipException
+from insights.parsers import calc_offset, keyword_search, optlist_to_dict, parse_delimited_table, parse_fixed_table,\
```

Comment:
I think `()` would be better here than `\`, it allows cleaner formatting of the imports.

Reply:
Ah good point, I updated it to that.